install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -90,6 +90,10 @@ spec:
         - name: KUBE_CACHE_MUTATION_DETECTOR
           value: "true"
 {{- end }}
+        # Set the TCP port for the agent health status API. This is not the port used
+        # for cilium-health.
+        - name: CILIUM_AGENT_HEALTH_PORT
+          value: {{ .Values.agent.healthPort }}
```

Comment:
Ah, I forgot that it would be an option here when providing pointers on Slack.

If this is also in the configmap (generated from the same values.yaml definition) then these lines will be unnecessary as the agent will automatically propagate a configmap entry named like `agent-health-port` into a bound env variable `CILIUM_AGENT_HEALTH_PORT` and hence into the daemon `option.AgentHealthPort` option.